app/controllers/health/careplans_controller.rb

```diff
@@ -144,45 +146,51 @@ def self_sufficiency_assessment
 
     def set_careplan
       @careplan = careplan_source.find(params[:id].to_i)
+      @cha = @careplan.patient.recent_cha_form
     end
 
     def set_epic_goals
       @epic_goals = @patient&.epic_goals&.visible
     end
 
     def careplan_params
+      careplan_params = [
```

Comment:
Maybe this should get moved to the care plan class so it can vary with versions/time as necessary?